packages/react/src/global-alert/GlobalAlert.test.tsx

```diff
@@ -47,12 +47,15 @@ describe('GlobalAlert', () => {
 
 	it('responds to an onDismiss event', async () => {
 		const onDismiss = jest.fn();
-		renderGlobalAlert({
+		const { container } = renderGlobalAlert({
 			title: 'Alert title',
 			children: <Text as="p">Alert description.</Text>,
 			onDismiss,
 		});
-		await userEvent.click(screen.getByText('Dismiss'));
+
+		expect(container).toMatchSnapshot();
```

Comment:
This snapshot shouldn't be part of the test "responds to an onDismiss event".

Can we remove it or move to it's own test?